woocommerce/woocommerce
# Add code snippet to change the number of related products output

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

Closes https://github.com/woocommerce/woodocs/issues/72.

This PR adds the code snippet [Change number of related products output](https://woocommerce.com/document/change-number-of-related-products-output/) to the Woo docs.

Apart from adding the new code snippet, I also applied the following minor adjustments to the file `docs/snippets/before-login--register-form.md`:
- Remove the tracking code from the link http://www.woocommerce.com/support-policy/.
- Add missing fullstops.

### How to test the changes in this Pull Request:

1. Compare the content of the file `docs/snippets/number-of-products-per-row.md` with the content on https://woocommerce.com/document/change-number-of-related-products-output/.
2. Verify that the new link within `docs/snippets/README.md` points to the newly added code snippet.
